assets/js/app.js

```diff
@@ -29,37 +29,37 @@ return t=c(t),t&&t.replace(jt,h).replace(Mt,"")}function fr(t,e,n){t=c(t),e+="";
+"use strict";!function(r,s){i=[n(298)],o=function(t){return s(t)}.apply(e,i),!(void 0!==o&&(t.exports=o))}(void 0,function(t){var e=function(){return{isMsie:function(){return/(msie|trident)/i.test(navigator.userAgent)?navigator.userAgent.match(/(msie |rv:)(\d+(.\d+)?)/i)[2]:!1},isBlankString:function(t){return!t||/^\s*$/.test(t)},escapeRegExChars:function(t){return t.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g,"\\$&")},isString:function(t){return"string"==typeof t},isNumber:function(t){return"number"==typeof t},isArray:t.isArray,isFunction:t.isFunction,isObject:t.isPlainObject,isUndefined:function(t){return"undefined"==typeof t},isElement:function(t){return!(!t||1!==t.nodeType)},isJQuery:function(e){return e instanceof t},toStr:function(t){return e.isUndefined(t)||null===t?"":t+""},bind:t.proxy,each:function(e,n){function i(t,e){return n(e,t)}t.each(e,i)},map:t.map,filter:t.grep,every:function(e,n){var i=!0;return e?(t.each(e,function(t,o){return(i=n.call(null,o,t,e))?void 0:!1}),!!i):i},some:function(e,n){var i=!1;return e?(t.each(e,function(t,o){return(i=n.call(null,o,t,e))?!1:void 0}),!!i):i},mixin:t.extend,identity:function(t){return t},clone:function(e){return t.extend(!0,{},e)},getIdGenerator:function(){var t=0;return function(){return t++}},templatify:function(e){function n(){return String(e)}return t.isFunction(e)?e:n},defer:function(t){setTimeout(t,0)},debounce:function(t,e,n){var i,o;return function(){var r,s,a=this,l=arguments;return r=function(){i=null,n||(o=t.apply(a,l))},s=n&&!i,clearTimeout(i),i=setTimeout(r,e),s&&(o=t.apply(a,l)),o}},throttle:function(t,e){var n,i,o,r,s,a;return s=0,a=function(){s=new Date,o=null,r=t.apply(n,i)},function(){var l=new Date,c=e-(l-s);return n=this,i=arguments,0>=c?(clearTimeout(o),o=null,s=l,r=t.apply(n,i)):o||(o=setTimeout(a,c)),r}},stringify:function(t){return e.isString(t)?t:JSON.stringify(t)},noop:function(){}}}(),n=function(){function t(t){var s,a;return a=e.mixin({},r,t),s={css:o(),classes:a,html:n(a),selectors:i(a)},{css:s.css,html:s.html,classes:s.classes,selectors:s.selectors,mixin:function(t){e.mixin(t,s)}}}function n(t){return{wrapper:'<span class="'+t.wrapper+'"></span>',menu:'<div class="'+t.menu+'"></div>'}}function i(t){var n={};return e.each(t,function(t,e){n[e]="."+t}),n}function o(){var t={wrapper:{position:"relative",display:"inline-block"},hint:{position:"absolute",top:"0",left:"0",borderColor:"transparent",boxShadow:"none",opacity:"1"},input:{position:"relative",verticalAlign:"top",backgroundColor:"transparent"},inputWithNoHint:{position:"relative",verticalAlign:"top"},menu:{position:"absolute",top:"100%",left:"0",zIndex:"
```

Comment:
@MarkJepson its still on the same line 32, must have removed some trailing whitespace